lib/class-wp-rest-block-types-controller.php

```diff
@@ -353,15 +353,15 @@ public function get_item_schema() {
 				),
 				'description'   => array(
 					'description' => __( 'Description of block type.', 'gutenberg' ),
-					'type'        => 'string',
-					'default'     => '',
+					'type'        => array( 'string', 'null' ),
+					'default'     => null,
```

Comment:
I think it could be fine, but I'm not quite as adamant that `description` couldn't always be a string, even an empty one. Unlike other fields like `icon` and `category`, there's less a risk of ambiguity between the empty string of a description and the finite set of options that an `icon` or `category` string can correspond to, or how a string asset value could be interpreted as anything that's valid to provide to `wp_register_script`.